### ISSUE TITLE
shaman thunderstorm rarely ever works

### ISSUE THREAD

**Issue by oldspices** [gamefreedomgit/Maelstrom#4817]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
it seems you have to be at the ABSOLUTE PERFECT ANGLE for it to work properly... even a 1 degree slope will cause it to fail, or something... no clue but id say it works about 5% of the time. 
**How to reproduce:**
just use it in pvp and watch it not do anything 

**How it should work:**
should really YEET people away. 

**Database links:**

**Comment by Hermeleen** [comment]
Maybe its spawns too high? if it there was a line between be the thunderstorm origin point and the player, if the origin of thunderstorm was too high, the force applied in direction towards the player would basically be forcing him to the ground. I know what you mean,  slight bump and it don't do nothing.

**Comment by Crusaderz** [comment]
Same for druid typhoon

**Comment by SimonBpvp** [comment]
I'm not sure from the post if youre talking about the gradient or something else so apologies if I misunderstood. I think with thunderstorm it's supposed to barely knock them any distance at all if you're using it uphill but knock them really extra far if you're knocking someone downhill. So I don't know if what you're describing is a bug that I'm not aware of or is like I described and is working as intended.

**Comment by Hermeleen** [comment]
> I'm not sure from the post if youre talking about the gradient or something else so apologies if I misunderstood. I think with thunderstorm it's supposed to barely knock them any distance at all if you're using it uphill but knock them really extra far if you're knocking someone downhill. So I don't know if what you're describing is a bug that I'm not aware of or is like I described and is working as intended.

yes, under ideal conditions. but OP said  " even a 1 degree slope will cause it to fail ". I saw what he means many times. Imagine a flat land, and somehow you happen to stand on little bump, even tho it should most likely "slide you" from the bump, it's feels like you got pushed against wall.

**Comment by oldspices** [comment]
https://ptb.discord.com/channels/1040715551531618365/1049306466949017690/1165822845587304448   here i posted a video of it clearly just not working when it should. if i was recording more i coud have endless videos lol it happens so often

**Comment by oldspices** [comment]
need to add to this that it seems to have something messed up with hit rating- i have over 10% hit and it gets resists a lot of the time, which should not be the case ever

**Comment by Hermeleen** [comment]
Just to say it, hit rating for spells is i believe 4% in pvp, maybe you mean spell penetration. That's the one for "resist" message.

**Comment by oldspices** [comment]
i have well over spell pen cap and hit cap and its resisting on players with 0 buffs whatsoever.   

also i was hearing its not knocking people who are stunned / poly etc  and its supposed to

**Comment by Afgann** [comment]
fixed
